fabrics/helpers/casadiFunctionWrapper.py

```diff
@@ -67,6 +67,14 @@ def evaluate(self, **kwargs):
                 for j, radius_obst_dyn_j in enumerate(kwargs[key]):
                     radius_dyn_dictionary[f'radius_obst_dynamic_{j}'] = radius_obst_dyn_j
                 argument_dictionary.update(radius_dyn_dictionary)
+            if key == 'radius_body_panda_links' or key == 'radius_body_panda_link':
+                """ The syntax should be: [A list of radii of the bodie, a list of the body numbers]
+                # Example: radius_body_panda_links=[[0.02, 0.02, 0.2], [3, 4, 9]] """
+                radius_body_dictionary = {}
+                for j, radius_body_j in enumerate(kwargs[key][0]):
+                    link_nr = kwargs[key][1][j]
+                    radius_body_dictionary[f'radius_body_panda_link{link_nr}'] = radius_body_j
```

Comment:
This is specific to the `panda` and does not generalize to other robots. Is there a way to make in generic?

Reply:
I am not super sure. This feature is not useful for pointmasses (but then it is just ignored), only for kinematic chains/arms. 
Then the whole input naming into compute_action() should be more logical for the radius of the bodies. That might be quite a big change to do for all examples. In boxer.py it is radius_body_ee_link for example. 
I think for now this feature would still be nice to have with radius_body_panda_links and it is just ignored in all other cases.

Reply:
The point is, that this implementation would not work for the ur5 arm for example. However, this should be feasible. You could for example access all the arguments with a leading 'radius' to extract the arguments for body links.